change/@fluentui-react-popover-0f00224e-dd41-401d-b085-6c1f1700cb91.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31108&commitId=309693287974f8e6ef810afc6eb70c86216c05c8&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|2377| Changed|

</details>


  <details><summary><b>Popover Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|21| Changed|
|Popover Converged.positioning.positioned popovers.chromium.png|11| Changed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|315| Changed|

</details>


  <details><summary><b>Positioning</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.arrow - RTL.chromium.png|1381| Changed|
|Positioning.arrow.chromium.png|1386| Changed|

</details>


  <details><summary><b>Tooltip Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip Converged.positioning - High Contrast.positioned tooltips.chromium.png|232| Changed|
|Tooltip Converged.positioning - RTL.positioned tooltips.chromium.png|27| Changed|
|Tooltip Converged.positioning.positioned tooltips.chromium.png|27| Changed|
|Tooltip Converged.inverted withArrow.chromium.png|22| Changed|
|Tooltip Converged.inverted withArrow - Dark Mode.chromium.png|3| Changed|
|Tooltip Converged.withArrow - High Contrast.chromium.png|21| Changed|
|Tooltip Converged.overflow-wrap withArrow.chromium.png|3| Changed|
|Tooltip Converged.withArrow.chromium.png|3| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>